File: twiml/voice/transcription/transcription-hints-tokens/transcription-hints-tokens.py
from twilio.twiml.voice_response import VoiceResponse, Start, Transcription

response = VoiceResponse()
start = Start()
start.transcription(
    status_callback_url='https://example.com/your-callback-url',
    hints='$OOV_CLASS_ALPHANUMERIC_SEQUENCE')
response.append(start)

print(response)

File: twiml/voice/transcription/transcription-hints-tokens/transcription-hints-tokens.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;

const response = new VoiceResponse();
const start = response.start();
start.transcription({statusCallbackUrl: 'https://example.com/your-callback-url', hints: '$OOV_CLASS_ALPHANUMERIC_SEQUENCE'});

console.log(response.toString());
